<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditUser</class>
 <widget class="QWidget" name="EditUser">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>140</y>
     <width>1500</width>
     <height>800</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#groupBox_2{background: #FFFFFF;
border-radius: 8px;}</string>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>110</y>
      <width>300</width>
      <height>565</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">width: 300px;
height: 565px;
background: #FFFFFF;
border-radius: 8px;</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="portrait_Label">
     <property name="geometry">
      <rect>
       <x>105</x>
       <y>123</y>
       <width>90</width>
       <height>90</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="name_LineEdit">
     <property name="geometry">
      <rect>
       <x>35</x>
       <y>283</y>
       <width>230</width>
       <height>50</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>黑体</family>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">width: 230px;
height: 50px;
background: #EEEEEE;
border-radius: 4px;
border: 1px solid #979797;</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="placeholderText">
      <string>请输入姓名</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="ID_LineEdit">
     <property name="geometry">
      <rect>
       <x>35</x>
       <y>353</y>
       <width>230</width>
       <height>50</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>黑体</family>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">width: 230px;
height: 50px;
background: #EEEEEE;
border-radius: 4px;
border: 1px solid #979797;</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
     <property name="placeholderText">
      <string>ID号</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="allLimp_Btn">
    <property name="geometry">
     <rect>
      <x>1130</x>
      <y>300</y>
      <width>80</width>
      <height>44</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background: #FFFFFF;
border-radius: 4px;
border: 1px solid #333300;
color:#333333;</string>
    </property>
    <property name="text">
     <string>双侧</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>959</x>
      <y>186</y>
      <width>150</width>
      <height>28</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>用户体重：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="weight_LineEdit">
    <property name="geometry">
     <rect>
      <x>1109</x>
      <y>175</y>
      <width>230</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background: #EEEEEE;
border-radius: 4px;
border: 1px solid #999999;</string>
    </property>
    <property name="placeholderText">
     <string>kg</string>
    </property>
   </widget>
   <widget class="QPushButton" name="rightLimp_Btn">
    <property name="geometry">
     <rect>
      <x>1020</x>
      <y>300</y>
      <width>80</width>
      <height>44</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background: #FFFFFF;
border-radius: 4px;
border: 1px solid #333300;
color:#333333;</string>
    </property>
    <property name="text">
     <string>右侧</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="male_RadioButton">
    <property name="geometry">
     <rect>
      <x>1130</x>
      <y>110</y>
      <width>80</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>男</string>
    </property>
    <property name="iconSize">
     <size>
      <width>22</width>
      <height>22</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="leftLimp_Btn">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>300</y>
      <width>80</width>
      <height>44</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background: #FFFFFF;
border-radius: 4px;
border: 1px solid #333300;
color:#333333;</string>
    </property>
    <property name="text">
     <string>左侧</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>509</x>
      <y>175</y>
      <width>150</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>用户身高：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>659</x>
      <y>300</y>
      <width>210</width>
      <height>44</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>上肢功能障碍：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>509</x>
      <y>110</y>
      <width>151</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>用户年龄：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>959</x>
      <y>110</y>
      <width>150</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>用户性别：</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="female_RadioButton">
    <property name="geometry">
     <rect>
      <x>1240</x>
      <y>110</y>
      <width>80</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>女</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="age_LineEdit">
    <property name="geometry">
     <rect>
      <x>659</x>
      <y>110</y>
      <width>230</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background: #EEEEEE;
border-radius: 4px;
border: 1px solid #999999;</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
    <property name="placeholderText">
     <string>岁</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="height_LineEdit">
    <property name="geometry">
     <rect>
      <x>659</x>
      <y>175</y>
      <width>230</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background: #EEEEEE;
border-radius: 4px;
border: 1px solid #999999;</string>
    </property>
    <property name="placeholderText">
     <string>cm</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>509</x>
      <y>240</y>
      <width>150</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>用户信息：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="userMsg_LineEdit">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>659</x>
      <y>240</y>
      <width>680</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background: #EEEEEE;
border-radius: 4px;
border: 1px solid #999999;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="remark_LineEdit">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>500</y>
      <width>680</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background: #EEEEEE;
border-radius: 4px;
border: 1px solid #999999;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="guardian_LineEdit">
    <property name="geometry">
     <rect>
      <x>1110</x>
      <y>365</y>
      <width>230</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background: #EEEEEE;
border-radius: 4px;
border: 1px solid #999999;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>922</x>
      <y>365</y>
      <width>181</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>监护人电话：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>365</y>
      <width>150</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>用户电话：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="address_LineEdit">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>440</y>
      <width>680</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background: #EEEEEE;
border-radius: 4px;
border: 1px solid #999999;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>500</y>
      <width>150</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>备注信息：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>440</y>
      <width>150</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>家庭住址：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="patientPhone_LineEdit">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>365</y>
      <width>230</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background: #EEEEEE;
border-radius: 4px;
border: 1px solid #999999;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="delete_Btn">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>700</y>
      <width>140</width>
      <height>70</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>28</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{background: #F78759;
border-radius: 8px;
color: rgb(255, 255, 255);}
QPushButton:hover{background: #F4673C;
border-radius: 8px;
color: rgb(255, 255, 255);}
QPushButton:pressed{background: #F26036;
border-radius: 8px;
color: rgb(255, 255, 255);}</string>
    </property>
    <property name="text">
     <string>删除</string>
    </property>
   </widget>
   <widget class="QPushButton" name="confirm_Btn">
    <property name="geometry">
     <rect>
      <x>1270</x>
      <y>700</y>
      <width>140</width>
      <height>70</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>28</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{background: #0D9DDB;
border-radius: 8px;
color: rgb(255, 255, 255);}
QPushButton:hover{background: #0A90D4;
border-radius: 8px;
color: rgb(255, 255, 255);}
QPushButton:pressed{background: #0984CE;
border-radius: 8px;
color: rgb(255, 255, 255);}</string>
    </property>
    <property name="text">
     <string>确认</string>
    </property>
   </widget>
   <widget class="QPushButton" name="save_Btn">
    <property name="geometry">
     <rect>
      <x>1270</x>
      <y>700</y>
      <width>140</width>
      <height>70</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>28</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background: #0D9DDB;
border-radius: 8px;
color:white;
</string>
    </property>
    <property name="text">
     <string>保存</string>
    </property>
   </widget>
   <widget class="QPushButton" name="modify_Btn">
    <property name="geometry">
     <rect>
      <x>1050</x>
      <y>700</y>
      <width>140</width>
      <height>70</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>28</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{background: #0D9DDB;
border-radius: 8px;
color: rgb(255, 255, 255);}
QPushButton:hover{background: #0A90D4;
border-radius: 8px;
color: rgb(255, 255, 255);}
QPushButton:pressed{background: #0984CE;
border-radius: 8px;
color: rgb(255, 255, 255);}</string>
    </property>
    <property name="text">
     <string>修改</string>
    </property>
   </widget>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1500</width>
      <height>80</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#widget{background: #0D9DDB;
border-top-left-radius: 8px;border-top-right-radius:8px}</string>
    </property>
    <widget class="QPushButton" name="close_Btn">
     <property name="geometry">
      <rect>
       <x>1430</x>
       <y>20</y>
       <width>40</width>
       <height>40</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/new/prefix1/release/source/image/close.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="title_Label">
     <property name="geometry">
      <rect>
       <x>560</x>
       <y>20</y>
       <width>411</width>
       <height>45</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>黑体</family>
       <pointsize>30</pointsize>
      </font>
     </property>
     <property name="text">
      <string>用户信息</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
